.github/workflows/build.yml

```diff
@@ -115,6 +116,69 @@ jobs:
           leeway build dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}"
           echo "previewctl_hash=$(leeway describe dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}" -t '{{ .Metadata.Version }}')" >> $GITHUB_OUTPUT
 
+  update_pr_desc:
+    name: Update PR summary
+    needs: [ configuration ]
+    if: needs.configuration.outputs.pr_number != ''
+    concurrency:
+      group: ${{ github.head_ref || github.ref_name }}-update-pr-desc
+    runs-on: [ self-hosted ]
+    container:
+        image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:kylos101-prebuild-jam-gha.5832
+        volumes:
+          - /var/tmp:/var/tmp
+          - /tmp:/tmp
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configure workspace
```

Comment:
i copied this step and container from previous job not sure whether it is really necessary

Reply:
Technically we don't need any of those, as the `previewctl report` is pretty static. You can remove the container/checkout/configure/set outputs and just have a template with the `preivewctl report` and substitute the values with the current branch name. If the install job succeeded, then they will be valid